### PR TITLE
[UXE-5417] feat: add SSO Management menu option and route guard to users that don't have federated_auth flag

### PR DESCRIPTION
**Pull request by aloisio-m-bastian**
### What is the new behavior introduced by this PR?
Add SSO Management option to profile menu and add route guard to block users that dont have access
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [x] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

**Review by lucasmendes21 — DISMISSED** [review]

File: src/router/hooks/guards/SSOManagementGuard.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/** @type {import('vue-router').NavigationGuardWithThis} */
+
+export function SSOManagementGuard({ to, accountStore }) {
+  if (to.name === 'identity-providers') {
+    if (!accountStore.hasAccessToSSOManagement) {
+      return '/'
+    }
+  } else {
```
Comment (pauloSF0) [inline src/router/hooks/guards/SSOManagementGuard.js:8]: não precisa de else, só fazer um `return true`

File: src/templates/profile-block/index.vue [hunk 2]
```diff
@@ -331,7 +331,10 @@
     {
       label: 'Teams Permissions',
       to: '/teams-permission'
-    }
+    },
+    ...(hasAccessToSSOManagement.value
```
Comment (pauloSF0) [inline src/templates/profile-block/index.vue:335]: Essa parte aqui, não está legal não ta dificil de entender, não seria melhore criar um method, exemplo:  
`
checkHasAccessToSSOManagement = () => {
 if(hasAccessToSSOManagement.value) {
      profile.value.push({
         label: 'SSO Management', to: '/identity-providers'
      })
   }
}
` e esse method ser chamado em uma onMounted, assim ficaria bem melhor de entender

Comment (aloisio-m-bastian) [reply src/templates/profile-block/index.vue:335]: não vejo pq criar mais um metodo e ter que usar onMount só para isso

File: src/router/hooks/guards/SSOManagementGuard.js (hunk 1 above)
Comment (aloisio-m-bastian) [reply src/router/hooks/guards/SSOManagementGuard.js]: feito

**Review by HerbertJulio — APPROVED** [review]

**Review by peterpaulo-azion — APPROVED** [review]

**Pull request merged**